core/templates/components/state-editor/state-editor-properties-services/state-editor.service.ts

```diff
@@ -319,6 +320,14 @@ export class StateEditorService {
     this.solutionValidityService.deleteSolutionValidity(this.activeStateName);
   }
 
+  setInitActiveContentId(newActiveContentId: string | null): void {
```

Comment:
newActiveContentId will be null when we are not visiting translation tab from the link in translation-review-modal. It is set to null to avoid any breakages in workflow of exploration-editor page.